### DISCUSSION TITLE
[projects beta] user tag in open issue comment

### DISCUSSION THREAD

**Discussion by mr13tech** [community/community#18609] [Projects and Issues]
Problem:

On open issue view, from projects beta, in comment - not all users visible in tag suggestion;

on 100% scale in browser only first 4 suggestions are visible.

Browser:  Firefox: `100.0.2 (64-bit`

Screenshot:

![tag-comments-blured](https://user-images.githubusercontent.com/29409876/173662892-08d33f12-5c4e-4772-8bf8-16840ac2d894.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
